### PR TITLE
Connor initial commit. Edited readme.

### PR DESCRIPTION
**Pull request by ConnorRogstad**

### CODE REVIEW

**Review by EllaMcMillan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# FlatCap_Hanoi
\ No newline at end of file
+# FlatCap_Hanoi
+
+
+## Author Information
+- Names: Connor Rogstad,
```
Comment (redacial) [inline README.md:5]: Cool Name

**Review by redacial — APPROVED** [review]
Well done!

**Comment by hooiyumcha** [comment]
ok

**Review by hooiyumcha — APPROVED** [review]
ok2

**Pull request merged**